### DISCUSSION TITLE
Feedback for Indicator Panel

### DISCUSSION THREAD

**Discussion by jessicaaustin** [foxglove/community#284] [Q&A]
Follow up to https://github.com/foxglove/studio/pull/2003

First of all, our team is really excited to use this new panel! We played around with it and this is our initial feedback:

* For colorblind users, it would be nice to have an indication other than color. For example, :heavy_check_mark: for OK and :x: for not OK, or different background patterns. In the meantime we can use Label to indicate this so not a huge deal
* It would be nice to show the actual data somewhere in the panel. For example, showing the actual data value instead of or in addition to a Label. This way, we can put everything in one panel
* Handle wrapping for small panels. Example: ![Selection_999(861)](https://user-images.githubusercontent.com/1159930/176065250-7163d856-9ebf-405b-960e-7b08e3aa468d.png)

So for example, imagine you wanted to monitor some velocity component, and alert if it went over some limit.

So instead:

| Raw Message Panel | Indicator Panel |
|-----------------------------|----------------------|
|  `/velocity.x`: 0.5        | :green_circle: Velocity OK       | 
|  `/velocity.x`: 1.5        | :red_circle: Velocity OVER LIMIT        | 

it would be nice to have:

| Indicator Panel |
|----------------------|
| :heavy_check_mark: Velocity OK: 0.5  | 
| :x: Velocity OVER LIMIT: 1.5  | 

(don't worry too much about formatting here, just trying to give an idea)

Thanks!

**Comment by jtbandes** [comment]
Related:
- foxglove/studio#3808
- foxglove/studio#4176 
- foxglove/community#263
